src/snowflake/snowpark/modin/plugin/extensions/io_overrides.py

```diff
@@ -146,7 +145,14 @@ def json_normalize(
 ) -> pd.DataFrame:  # noqa: PR01, RT01, D200
```

Comment:
do we need to remove the noqa, since we now support this and want it type checked?

Reply:
these #noqa marks are actually for lack of docstrings (PR01 is parameters, RT01 is return value, I forget what D200 is), which I think we're OK with because we define it elsewhere